input/_ExtensionsList.cshtml

```diff
@@ -40,16 +43,17 @@
                     <li>
                         Latest version: @version
                     </li>
-                    <li>
-                        <i class="fa fa-github"></i>
-                        <a href="@repository" target="_blank">@repository.Replace("https://github.com/", string.Empty).TrimEnd('/')</a>
-                        @if (repository.StartsWith("https://github.com/cake-contrib/"))
-                        {
-                            <a href="/community/cake-contrib">
-                                <i class="fa fa-check-circle" data-toggle="tooltip" title="Source code hosted in cake-contrib organization"></i>
-                            </a>
-                        }
-                    </li>
+                    @if (!string.IsNullOrWhiteSpace(publishDate))
+                    {
+                        var parsedPublishDate = @DateTime.Parse(publishDate, null, System.Globalization.DateTimeStyles.RoundtripKind);
+
+                        <li>
+                            <span title="@parsedPublishDate.ToOrdinalWords()">
```

Comment:
Does CultureInfo (or similar) is needed here to force English just in case? The `Oktober` in the tooltip gives me the impression it's using your machine's settings (prob. not an issue in the build server, but thought I'd point it out)

Reply:
Good catch. I fixed the whole site to `en-GB` culture.